lib/admin/WP_Auth0_Admin_Features.php

```diff
@@ -378,21 +373,28 @@ public function render_override_wp_avatars( $args = array() ) {
 	}
 
 	public function basic_validation( $old_options, $input ) {
-		// SLO will be turned off in WP_Auth0_Admin_Features::sso_validation() if SSO is not on.
-		$input['singlelogout']        = ( isset( $input['singlelogout'] ) ? $input['singlelogout'] : 0 );
-		$input['override_wp_avatars'] = ( isset( $input['override_wp_avatars'] ) ? $input['override_wp_avatars'] : 0 );
+		$input['sso'] = ! empty( $input['sso'] ) ? 1 : 0;
```

Comment:
Adding in validation from removed `sso_validation` middleware.

Reply:
Maybe flip the inversion and results to simplify?   `empty( $input['sso'] ) ? 0 : 1;`